lisa/combinators/batch_combinator.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Again, docs? What is it? Matching expect list? Something like that?

Reply:
added doc